upgrade/run_upgrade_test.sh

```diff
@@ -38,48 +38,55 @@ export TARGET_RELEASE_PATH=${TAGET_RELEASE_PATH:-"https://storage.googleapis.com
 export INSTALL_OPTIONS=${INSTALL_OPTIONS:-"helm"}
 export FROM_PATH=${FROM_PATH:-"$(mktemp -d from_dir.XXXXXX)"}
 export TO_PATH=${TO_PATH:-"$(mktemp -d to_dir.XXXXXX)"}
-export LINUX_TAR_SUFFIX=${LINUX_TAR_SUFFIX:-"linux-amd64.tar.gz"}
+export SOURCE_LINUX_TAR_SUFFIX=${SOURCE_LINUX_TAR_SUFFIX:-"linux-amd64.tar.gz"}
+export TARGET_LINUX_TAR_SUFFIX=${TARGET_LINUX_TAR_SUFFIX:-"linux-amd64.tar.gz"}
 
 function get_git_sha() {
   local url_path=${1}
   local tag=${2}
 
-  GIT_SHA=""
+  GIT_SHA=
+  LINUX_TAR_SUFFIX=
 
   if [[ "${tag}" =~ "latest" ]];then
     release_version=$(echo "${tag}" | cut -d'_' -f1)
     # shellcheck disable=SC2072
     if [[ ${release_version} < "1.6" ]]; then
-       export LINUX_TAR_SUFFIX="linux.tar.gz"
+       LINUX_TAR_SUFFIX="linux.tar.gz"
```

Comment:
don't you need to echo this var to make it available globally?

Reply:
Shell variables by default are globally scoped.